ci/trim_pipeline.py

```diff
@@ -25,7 +25,7 @@ def main() -> int:
         return 0
 
     # Otherwise, filter down to the selected tests.
-    with open(Path(__file__).parent / "pipeline.yml") as f:
+    with open(Path(__file__).parent / sys.argv[1] / "pipeline.yml") as f:
```

Comment:
Could you switch this to `argparse`? I've been trying to build the habit of doing that for even these really simple scripts. The error messages with `argparse` are just _so_ much better if someone mis-invokes the script.

```
import argparse

parser = argparse.ArgumentParser()
parser.add_argument("pipeline")
args = parser.parse_args()

with open(Path(__file__).parent / args.pipeline / "pipeline.yml"
```